Formula/binutils.rb

```diff
@@ -36,27 +34,25 @@ def install
       "--enable-interwork",
       "--enable-multilib",
       "--enable-64-bit-bfd",
+      "--enable-gold",
       "--enable-plugins",
       "--enable-targets=all",
       "--with-system-zlib",
       "--disable-nls",
-      "--disable-gold",
-      "--disable-gprofng", # Fails to build on Linux
     ]
     system "./configure", *args
-    # Pass MAKEINFO=true to disable generation of HTML documentation.
-    # This avoids a build time dependency on texinfo.
-    make_args = OS.mac? ? [] : ["MAKEINFO=true"]
-    system "make", *make_args
-    system "make", "install", *make_args
+    system "make"
+    system "make", "install"
 
     if OS.mac?
       Dir["#{bin}/*"].each do |f|
         bin.install_symlink f => "g" + File.basename(f)
       end
     else
       # Reduce the size of the bottle.
-      system "strip", *bin.children, *lib.glob("*.a")
+      bin_files = bin.children
+      bin_files -= [bin/"gp-display-html"] # This is not a binary executable.
```

Comment:
```suggestion
      bin_files = bin.children.select(&:elf?)
```
Something like this should work.

Reply:
It worked great!  That's a really neat solution if we run into this problem again.